tests/test_e2e_60_of_multi_table.py

```diff
@@ -0,0 +1,430 @@
+
+        # Add pipeline
+        api_url = f"{KYTOS_API}{OF_MULTI_TABLE_API}"
+        response = requests.post(api_url, json=pipeline)
+        data = response.json()
+        assert response.status_code == 201
+        assert 'id' in data
+
+        # Enabled pipeline
+        api_url = f"{KYTOS_API}{OF_MULTI_TABLE_API}/{data['id']}/enable"
+        response = requests.post(api_url)
+        assert response.status_code == 200
+        time.sleep(10)
+
+        s1 = self.net.net.get('s1')
+        flows_s1 = s1.dpctl('dump-flows')
+        assert 'actions=resubmit(,1)' in flows_s1
+
+        # Delete all flows from switch 1
+        s1.dpctl('del-flows')
+        self.net.reconnect_switches()
+
+        time.sleep(10)
+
+        flows_s1 = s1.dpctl('dump-flows')
+        assert 'actions=resubmit(,1)' in flows_s1
+
+    def test_015_mef_eline_pipelined(self):
+        "Test if mef_eline flows can communicate through tables"
+        pipeline = {
+            "multi_table": [
+                {
+                    "table_id": 0,
+                    "description": "First table for miss flow entry",
+                    "table_miss_flow": {
+                        "priority": 0,
+                        "instructions": [{
+                            "instruction_type": "goto_table",
+                            "table_id": 1
+                        }],
+                        "match": {}
+                    },
+                },
+                {
+                    "table_id": 1,
+                    "description": "Second table for coloring",
+                    "napps_table_groups": {
+                        "mef_eline": ["epl", "evpl"]
+                    },
+                },
+            ]
+        }
+
+        evc = {
+            "name": "evc01",
+            "enabled": True,
+            "dynamic_backup_path": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+                "tag": {"tag_type": 1, "value": 100}
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:02:1",
+                "tag": {"tag_type": 1, "value": 100}
+            }
+        }
+
+        # Add circuit
+        api_url = f"{KYTOS_API}/mef_eline/v2/evc/"
+        response = requests.post(api_url, json=evc)
+        assert response.status_code == 201, response.text
+        data = response.json()
+        assert 'circuit_id' in data
+        time.sleep(10)
+
+        h11, h2 = self.net.net.get('h11', 'h2')
+        h11.cmd(f'ip link add link {h11.intfNames()[0]} name vlan_ra type vlan id 100')
+        h11.cmd('ip link set up vlan_ra')
+        h11.cmd('ip addr add 100.0.0.11/24 dev vlan_ra')
+        h2.cmd(f'ip link add link {h2.intfNames()[0]} name vlan_ra type vlan id 100')
+        h2.cmd('ip link set up vlan_ra')
+        h2.cmd('ip addr add 100.0.0.2/24 dev vlan_ra')
+        result = h11.cmd('ping -c1 100.0.0.2')
+        assert ', 0% packet loss,' in result
+
```

Comment:
Fantastic test case. For completeness though, could you assert that all flows are on table 0? You don't need to assert all individual flows since you covered that well on a prior test case, but let's ensure that at least it's not an empty collection and they all are on `table_id=0`, similarly, after enabling the pipeline and some delay, let's ensure `mef_eline` flows moved to `table_id=1`

Reply:
Asserting flows in table 0 and `mef_eline` flows in table 1 after enabling.